app/routes/_layout+/assets.$assetId.tsx

```diff
@@ -50,6 +51,14 @@ import { deleteAssets } from "~/utils/storage.server";
 
 export async function loader({ request, params }: LoaderFunctionArgs) {
   const { userId } = await requireAuthSession(request);
+  const organization = await getOrganizationByUserId({
```

Comment:
I dont think you need to do that. You can just adjust the query for asset by including organization and selecting only the currency field.
Something like this. 
```
include: {
  organization: {
    select: {
      currency: true
     },
  }
}
```

and then u can do `asset.organization.currency`